<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RejectConrInterview</class>
 <widget class="QWidget" name="RejectConrInterview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>702</width>
    <height>517</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>702</width>
    <height>517</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>702</width>
    <height>517</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reject Contractor Window</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вікно відмови контрагенту у роботі</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Оберіть контрагента:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="contr_reject_tb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>411</width>
     <height>192</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <column>
    <property name="text">
     <string>Прізвище</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ім'я</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Посада</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id_interview</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="cansel_bt">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Скасувати операцію</string>
   </property>
  </widget>
  <widget class="QPushButton" name="contr_reject_bt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Відхилити</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="search_contr_to_st_lb">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>211</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Пошук за прізвищем:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_contr_to_st_bt">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Пошук</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_contr_to_st_edit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_bt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Перезавантажити</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
